packages/frontend-api/src/Model/Mutation/Customer/User/Exception/InvalidAccountOrPasswordUserError.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Mutation\Customer\User\Exception;
+
+use GraphQL\Error\UserError;
+use Shopsys\FrontendApiBundle\Model\Error\UserErrorWithCodeInterface;
+
+class InvalidAccountOrPasswordUserError extends UserError implements UserErrorWithCodeInterface
```

Comment:
Does it make sense to throw this exception instead of `LoginFailedUserError` in `LoginMutation`?

Reply:
yes! :+1: a28452ac1